src/router.js

```diff
@@ -5,14 +5,19 @@ import Vue from "vue"
 /* istanbul ignore next */
 Vue.use(Router)
 
-export const routeGuard = store => (to, from, next) => {
+export const routeGuard = store => async (to, from, next) => {
   // Redirect if fullPath begins with a hash (fallback for old pre history mode urls)
   if (to.fullPath.includes("#")) {
     const path = to.fullPath.substr(to.fullPath.indexOf("#") + 1)
     next(path)
     return
   }
 
+  if (!(await featureAvailable(store, to))) {
+    next(`/feature-not-available/${to.meta.feature}`)
```

Comment:
Should this be lowerCase too for consistency?

Reply:
This was to pass the feature string that is actually displayed to the page. If we pass the lower case string, we would need to post process it in the page. Maybe could be done differently.

Reply:
We could use `this.$route.name` for for the title?

Reply:
`this.$route.name` actually should be lower case imo. It is used as an idea to describe routes. How would you pass `this.$route.name` to the feature page?

Reply:
That sounds reasonable.

I see now that `to.meta.feature` is used for more than one route, so we couldn't use the fields in reverse ways. So perhaps this is fine for now. We are optimising for the edge case, but I can see why.

(Perhaps each feature flag should have an associated title string, but it might be overkill for the moment.)